core/src/test/java/org/frankframework/pipes/IsUserInRolePipeTest.java

```diff
@@ -0,0 +1,304 @@
+package org.frankframework.pipes;
+
+import org.frankframework.configuration.ConfigurationException;
+import org.frankframework.core.ISecurityHandler;
+import org.frankframework.core.PipeForward;
+import org.frankframework.core.PipeLineSession;
+import org.frankframework.core.PipeRunResult;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+/**
+ * IsUserInRolePipe unit tests.
+ *
+ * @author <Laurens Mäkel>
+ */
+public class IsUserInRolePipeTest extends PipeTestBase<IsUserInRolePipe> {
+
+	public final String ROLE = "MyRole";
+	public final String ROLE2 = "MyRole1";
+	public final String ROLE3 = "MyRole2";
+
+	public final String NOT_IN_ROLE_FORWARD_NAME = "notInRole";
+	public final String NOT_IN_ROLE_FORWARD_PATH = "doSomething";
+
+	@Override
+	public IsUserInRolePipe createPipe() throws ConfigurationException {
+		return new IsUserInRolePipe();
+	}
+
+	@Test
+	public void requiresNotInRoleForward() {
+		assertThrows(ConfigurationException.class, () -> pipe.configure(), "notInRoleForwardName not found");
+	}
+
+	@Test
+	public void userIsInSingleRoleAttribute() throws Exception {
+		// Given
+		ISecurityHandler securityHandler = mock(ISecurityHandler.class);
+		PipeLineSession mockedSession = mock(PipeLineSession.class);
+		when(mockedSession.getSecurityHandler()).thenReturn(securityHandler);
+		when(securityHandler.isUserInRole(any(String.class))).thenReturn(true);
```

Comment:
Please put the repeated 3 or 4 lines of code in a new `@BeforeEach` method instead.